### ISSUE TITLE
Proposal: Completely disable "Discussions" within nodejs/node

### ISSUE THREAD

**Issue by ovflowd** [nodejs/admin#918]
It is no mystery that on the Node.js project, we use solely "issues" (within nodejs/node) to propose RFCs, new features, and general discussions surrounding the Node runtime.

Issues are also used for help requests, although people get redirected to nodejs/help for help with Node.js;

The nodejs/node repository has a discussions tab that is rarely used (very few threads are posted). It does feel like a duplication of nodejs/help and isn't being used for what it was intended to be used for.

Let's disable the Discussions tab altogether for "nodejs/node." That said, there are WGs or other projects within the Node.js org that pretty well use discussions, such as nodejs/collaborators, nodejs/nodejs.org, and others.

Although having a "Discussions" tab can be seen as a positive thing, it requires extra moderation; it is often just misused and, at the moment, doesn't bring anything to the table.

**Comment by ovflowd** [comment]
cc @nodejs/tsc

**Comment by aduh95** [comment]
In https://github.com/nodejs/admin/issues/771, the proposal was to migrate nodejs/help to Discussions, though that has not happened.

**Comment by ovflowd** [comment]
Welp, I believe that said issue should be superseded with this. We can indeed migrate those discussions as issues on nodejs/help, for any open discussion if needed and then disable discussions.

**Comment by gireeshpunathil** [comment]
IMO, `help repo` is not the duplicate of `discussions`, `nodejs` slack channel is.

**Comment by ovflowd** [comment]
> IMO, `help repo` is not the duplicate of `discussions`, `nodejs` slack channel is.

Have you read my thread at all? I explicitly said that current discussions are not used for discussions but help questions.

**Comment by ovflowd** [comment]
(I updated my prior comment adding some clarifications)

**Comment by gireeshpunathil** [comment]
so, if we want the discussions to be better discoverable within the org, `discussions` needs to be retained in lieu of the slack channel. or else it makes sense to shut it down.

I agree that currently the actual discussions get spread thin all over the place.

**Comment by gireeshpunathil** [comment]
only discussions pertinent to node/node are scoped under node/node discussions, right? so discussions happening in issues or discussions tab in other repos are fair?

either way, I agree with your proposal to shut down the discussions tab.

**Comment by targos** [comment]
Just remember that the discussions tab on nodejs/node is a redirect to the global discussions tab of the entire org.

**Comment by ovflowd** [comment]
> Just remember that the discussions tab on nodejs/node is a redirect to the global discussions tab of the entire org.

I don't think so. They are their own discussions tab, the only difference is they are marked as the default one.

That can be disabled too.

**Comment by aduh95** [comment]
Relevant discussion regarding nodejs/help vs "Discussions"" https://github.com/nodejs/admin/issues/802#issuecomment-1599035742

Relevant discussion regarding org discussion: https://github.com/nodejs/admin/issues/679#issuecomment-1460523085

**Comment by ovflowd** [comment]
> I tried it before posting my comment. They point to the same page.

Yes, but global discussions is not a thing -- you can configure to one of your repositories to be the default "discussions" tab that shows up on your org.

https://docs.github.com/en/organizations/managing-organization-settings/enabling-or-disabling-github-discussions-for-an-organization#enabling-or-disabling-github-discussions-for-your-organization

**Comment by jasnell** [comment]
+1 to disabling discussions.

**Comment by marco-ippolito** [comment]
+1 to disable, I never found particular value in them

**Comment by mcollina** [comment]
+1 to disable, I can't really follow any relevant discussions in them.

**Comment by BridgeAR** [comment]
+1 to disable discussions

**Comment by ovflowd** [comment]
I believe we got consensus here? Should we proceed then?

**Comment by marco-ippolito** [comment]
I think there is consensus and we can disable them

**Comment by ovflowd** [comment]
Did someone disable the feature? cc @nodejs/tsc

**Comment by targos** [comment]
> Discussions cannot be disabled because they are set as the source for the nodejs organization's public discussions.

**Comment by ovflowd** [comment]
> > Discussions cannot be disabled because they are set as the source for the nodejs organization's public discussions.

They can still be disabled if the org-wide feature gets disabled

**Comment by ovflowd** [comment]
@nodejs/tsc is someone going to do this?

**Comment by jasnell** [comment]
hmm, likely needs some other tweaking. Is there an alternative repo for us to direct the org discussions to?

![Image](https://github.com/user-attachments/assets/57a15da5-7cb3-40d6-9d18-4599b33e60f6)

**Comment by ovflowd** [comment]
I believe you need to update the org discussion settings and disable "org" discussions (repositories can still individually have their own discussions area) it's just that nodejs/node is marked as the home (default) discussion for Node org. But that's exactly what we want to get rid off :)

**Comment by ovflowd** [comment]
Monday we are releasing the Discord server too, on an unrelated note 🙈

**Comment by jasnell** [comment]
Ok, before making that change, let me just double check with @nodejs/tsc to make sure nothing has changed that alters the consensus here. If I see nothing by end of day Monday I'll make the switch.

**Comment by bjohansebas** [comment]
So, can it be disabled now?

**Comment by bjohansebas** [comment]
It feels strange to have both issues and discussions enabled at the same time in help repository. Which one should a user use if they need help? We should simplify the way to ask for help, not make it confusing.
